sonar-project.properties

```diff
@@ -3,6 +3,7 @@ sonar.host.url=https://sonar.flighttracker.online
 sonar.sourceEncoding=UTF-8
 sonar.sources=src
 sonar.exclusions=**/node_modules/**
+sonar.coverage.exclusions=**/src/environments/**,**/src/app/utils/**,**/src/lib/simulation-entities/**,**/src/lib/socket-events/**,**/src/lib/utils/**
```

Comment:
We can exclude `/src/environments/` but I am not sure it is a good idea to include the other paths from coverage since we can and should probably test them.

Reply:
I kinda just figured we want to exclude the src/lib/simulation-entities, src/lib/socket-events, and src/lib/utils because its all just data types, do we still want to keep those?